package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1035,11 +1035,13 @@ def principal_axes(group, pbc=False):
 
 
         .. versionchanged:: 0.8 Added *pbc* keyword
+        .. versionchanged:: 1.0.0 
+        Always return principal axes in right hand convention.
```

Comment:
Needs to be indented properly
```suggestion
           Always return principal axes in right hand convention.
```